cobc/dump_ast_gen.c

```diff
@@ -36,8 +36,8 @@ static void init_ptr_table ()
     ptr_table = (void**) calloc (ptr_table_size, sizeof (void*) );
     num_table = (int*)   calloc (ptr_table_size, sizeof (int) );
   } else {
-    bzero (ptr_table, ptr_table_size * sizeof(void*) );
-    bzero (num_table, ptr_table_size * sizeof(int) );
+    memset (ptr_table, 0, ptr_table_size * sizeof(void*) );
+    memset (num_table, 0, ptr_table_size * sizeof(int) );
```

Comment:
Is this a manual change after generation? In this case you possibly want to define bzero as macro instead to do that change.

Reply:
The new file is generated by a new version of the tool (https://github.com/OCamlPro/ctypes-gen), but I guess most devs will edit it by hand, and the tool will only be used from time to time to get a clean version.